ColorWheel.js

```diff
@@ -1,183 +1,181 @@
 // @flow
 
-import React, {Component} from 'react'
+import React, { Component } from "react";
 import {
   Animated,
   Image,
   Dimensions,
   PanResponder,
   StyleSheet,
-  View,
-} from 'react-native'
-import colorsys from 'colorsys'
+  View
+} from "react-native";
+import colorsys from "colorsys";
 
 export class ColorWheel extends Component {
   static defaultProps = {
     thumbSize: 50,
-    initialColor: '#ffffff',
+    initialColor: "#ffffff",
     onColorChange: () => {},
-    precision: 0,
-  }
+    precision: 0
+  };
```

Comment:
I think you need to review your eslint settings, or avoid autoformatting for this PR